Packs/JoeSecurity/ReleaseNotes/1_1_23.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Joe Security v2
+- Updated the Docker image to: *demisto/vendors-sdk:1.0.0.103334*.
+
+The ***file*** command DBotScore result will now have the SHA256 as the file indicator value instead of the file name
```

Comment:
```suggestion
Fixed an issue where the *DBotScore.Indicator* output of the ***file*** command contained the file name instead of the file's SHA256.
```